src/plugin-vite.js

```diff
@@ -66,6 +67,17 @@ export const Vueless = function (options = {}) {
         await createTailwindSafelist({ mode, env, debug, targetFiles });
       }
 
+      if ((config.command.includes("sb:") && mode === "storybook") || isVuelessEnv) {
+        try {
+          const { default: buildWebTypes } = await import("@vueless/storybook/webTypes");
+
+          await buildWebTypes(vuelessConfig);
```

Comment:
I think you can move all this code into `utils/webTypes.js` to make the plugin code more clear.